server/views/pages/trivy.njk

```diff
@@ -39,6 +40,18 @@
 {% block bodyEnd %}
   <script type="module" src="/assets/govukFrontendInit.js"></script>
   <script src="/assets/moj/all.js"></script>
+  <script src="/assets/mojFrontendInit.js"></script>
+  <script src="https://www.gstatic.com/charts/loader.js"></script>
+  <script src="/assets/js/dayjs/dayjs.min.js"></script>
+  <script src="/assets/js/dayjs/plugin/relativeTime.js"></script>
   <script src="/assets/js/common.js"></script>
   <script src="/assets/js/trivy.js"></script>
+<script nonce="{{ cspNonce }}">
+    jQuery(function () {
+      const components = {{components | toJson(2) | safe }}
```

Comment:
I was thinking.. another option would be to load up all the component data in `trivy.ts` and pass through here instead of just the names..

This would mean that we don't have the lag between loading the page and we could get rid of the /trivy/data endpoint and the API call which would simplify things quite a bit..